.github/workflows/integration-tests.yml

```diff
@@ -91,3 +91,25 @@ jobs:
             --file "scripts/docker-compose.yml" \
             down
         if: matrix.database-type != 'sqlite3' && matrix.database-type != 'better-sqlite3'
+
+  build:
```

Comment:
is this name not a duplicate with existing one?

Reply:
Good catch! Fixed and pushed.